applications/models.py

```diff
@@ -364,3 +377,16 @@ def __str__(self):
             f"user='{self.bootcamp_application.user.email}', run='{self.bootcamp_application.bootcamp_run.title}', "
             f"contenttype={self.content_type}, object={self.object_id}"
         )
+
+
+class ApplicantLetter(TimestampedModel):
+    """Represents a letter sent to an applicant, with a hash so the user can view the letter afterwards"""
+
+    is_acceptance = models.BooleanField()
```

Comment:
Maybe out of scope for this PR, but should waitlisted applicants get an email too?  If that will be implemented eventually, might want something other than a boolean to determine which email to send.

Reply:
Yeah, I'll change to a CharField